config.json

```diff
@@ -14,6 +14,16 @@
         "text formatting"
       ]
     },
+	{
+	  "slug": "isogram",
+	  "difficulty": 1,
+	  "topics": [
+	    "arrays",
+	    "strings",
+		"control-flow (if-else statements)",
+		"control-flow (loops)"
```

Comment:
Looks like you've mixed tabs and spaces. Be sure to always use spaces in order to maximize your income (https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/).